CHANGELOG

```diff
@@ -11,8 +11,6 @@ Qtile 0.x.x, released xxxx-xx-xx:
         - memory widget can now be displayed with decimal values
         - new "qtile migrate" command, which will attempt to upgrade previous
           configs to the current version in the case of qtile API breaks.
-    * bugfixes
-        - fix KeyboardLayout widget (xkb-switch is new dependency)
```

Comment:
we probably want to keep this

Reply:
What do you mean by this? If we drop the fix, shouldn't we drop the changelog note about it?

Reply:
Oh sorry, I thought it was released already. Makes much more sense to revert then.